PEM - 4311 corrections (#3314)
EnterpriseDB/docs: product_docs/docs/pem/8/considerations/setup_ha_using_efm.mdx

- /pem/latest/pem_ha_setup/setup_ha_using_efm/
---

Failover Manager is a high-availability tool from EDB that enables a Postgres primary node to failover to a standby node during a software or hardware failure on the primary.

> You can set up Failover Manager only for the Postgres Enterprise Manager (PEM) server with a fresh installation. You can't set it for an existing PEM installation.

Reviewer: Sorry, but this still reads weird to me a bit. How about"

```suggestion
You can set up Failover Manager only with a fresh installation of Postgres Enterprise Manager (PEM). You can't set it for an existing PEM installation.
```
If we need to specify that it is the PEM server:
```suggestion
You can set up Failover Manager only with a fresh installation of a Postgres Enterprise Manager (PEM) server. You can't set it for an existing PEM installation.
```

> You can only set up the Failover Manager for Postgres Enterprise Manager (PEM) server with the fresh installation. You can't set it for the existing PEM installation.

Reviewer 2: Also, please move the word "only" to precede what it's modifying: *only* for the Postgres...

Otherwise looks good!
